### PR TITLE
Tanmay's Code Review

### PR DESCRIPTION
**Pull request by tanmayjog**
Below are some of the changes I have made in your project

1. No need for two lines in index.cshtml. Removed redundant piece of code
2. Inserted table classes and tags to improve the look of the table
3. Removed an extra semicolon in file index.cshtml.cs on line 60
4. Removed initialization of string named downloadedJson in file index.cshtml.cs on line 57
5. I observed one of the owner's names was blank. Instead of keeping that variable null, I initialized to "No Information Available" and made the according changes in the HTML page to highlight similar missing variables

                                                 **Earlier Output**

Earlier_Output

                                                **New Output**
New_Output

### CODE REVIEW

**Comment by tanmayjog** [comment]
Q: Summarize the three technical changes you made, and why.
A: 
1.	No need for two lines in index.cshtml. Removed redundant piece of code

2.	Inserted table classes and tags to improve the look of the table

3.	Added comments to make code more legible in file index.cshtml.cs

4.	Removed an extra semicolon in file index.cshtml.cs on line 60

5.	Removed initialization of string named downloadedJson in file index.cshtml.cs on line 57

6.	I observed one of the owner's names was blank. Instead of keeping that variable null, I initialized to "No Information Available" and made the corresponding changes in the HTML page to highlight similar missing variables

Q: List three items you learned from the code review.
A: 
1. I learned to use the breakpoint feature effectively to follow the sequence of code
2. I learned the importance of adding comments explaining the code. Since there were no comments added, it was a bit difficult for a novice developer like me to understand why some piece of code is written in the way it is and what does that piece of code do
3. I learned many new GitHub features like creating a new feature branch, how to merge branches and creating pull requests. I was unaware of these basic features before this code review

Q: Was the project available in GitHub on time?
A: Yes

Q: Does the project compile?
A: Yes

Q: Does the project have adequate documentation?
A: No

File: CafeRecords/Pages/Index.cshtml [hunk 1]
```diff
@@ -2,30 +2,42 @@
...
-        </tr>
-        @{
-            foreach (OwnerRecord.Owner owner in cafeOwners)
-            {
-                <tr>
-                    <td>@owner.AccountNumber</td>
-                    <td>@owner.DoingBusinessAsName</td>
-                    <td>@owner.OwnerFirstName @owner.OwnerMiddleInitial @owner.OwnerLastName</td>
-                </tr>
+    <table class="table table-bordered table-striped">
```
Comment (discospiff) [inline CafeRecords/Pages/Index.cshtml:11]: Nice styling additon... looks like css or Boostrap.

File: CafeRecords/Pages/Index.cshtml [hunk 2]
```diff
@@ -2,30 +2,42 @@
...
+            @{
+                foreach (OwnerRecord.Owner owner in cafeOwners)
+                {
+                    <tr>
+                        <td>@owner.AccountNumber</td>
+                        <td>@owner.DoingBusinessAsName</td>
+                        <td>
+                            @if (owner.OwnerFirstName == "No Information Available")
+                            {
+                                <span class="font-weight-bolder text-danger">@owner.OwnerFirstName</span>
```
Comment (discospiff) [inline CafeRecords/Pages/Index.cshtml:29]: Good use of styling to notify about no first name.

File: CafeRecords/Pages/Index.cshtml.cs [hunk 3]
```diff
@@ -9,6 +9,13 @@
 using Microsoft.Extensions.Logging;
 using OwnerRecord;
 
+/*Cafe Owner Project Combines two JSON files.
+ * One file named Owners has the information about business owners and their titles
+ * Another file named Cafe has information like address, permit number, Issue dte etc about cafes present in Chicago area 
+ * The field common to both of these JSON is Account Number
+ * By matching the Account Number we can determine the owner name of the cafe and display the information in a table
+ */
+
```
Comment (discospiff) [inline CafeRecords/Pages/Index.cshtml.cs:18]: A nice documentation addition.  One suggestion: use the Microsoft Intellisense style for documentation:
///
Not JavaDoc:
/**
 */

File: CafeRecords/Pages/Index.cshtml.cs [hunk 4]
```diff
@@ -43,21 +52,33 @@ public void OnGet()
...
                 foreach (var cafe in cafes)
                 {
+                    //Compare Account Number from two JSON and select only ones which have common account number
                     if (cafe.Value.AccountNumber == owner.AccountNumber)
                     {
                         cafeOwners.Add(owner);
                     }
+
+                    if(owner.OwnerFirstName == null)
+                    {
```
Comment (discospiff) [inline CafeRecords/Pages/Index.cshtml.cs:62]: Good informative check.

**Comment by discospiff** [comment]
Many good suggestions here that reduce technical debt.  I recommend merging these changes.

**Pull request merged**